checker/checker_dartagnan.go

```diff
@@ -30,6 +30,8 @@ func init() {
 	tools.RegEnv("DARTAGNAN_SET_OPTIONS", "",
 		"Options passed to Dartagnan, replacing the default options")
 	tools.RegEnv("DARTAGNAN_CAT_PATH", "", "Path to custom .cat files")
+	tools.RegEnv("DARTAGNAN_METHOD", "eager", "Backend method (values: eager | lazy)")
+	tools.RegEnv("DARTAGNAN_BOUND", "", "Unroll bound integer (default unset)")
```

Comment:
In principle, it might be useful to also set a bound for genmc (at least for testing). Why not having a model-checker independent option that works for both?

Reply:
I prefer not to let the user get used with the bound in genmc. The right thing is to use no bound.

Reply:
I'd wait to see how things evolve.